app/src/db/migrations/20240705184109_006_statistics-function-update.ts

```diff
@@ -0,0 +1,210 @@
+import type { Knex } from 'knex';
```

Comment:
Please add `/* eslint-disable max-len */` as the first line in this file to prevent it from spewing warnings in this and future PRs.

Please do the same for the file 20240621180350_005_soft-delete-activity.ts

Reply:
done